recipes/manim/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "manim" %}
+{% set version = "0.15.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/manim-{{ version }}.tar.gz
+  sha256: ba851ac5e877214c4aae97f28bffe483efe97cb68159480897e13e0b9e9a86a4
+
+build:
+  skip: true  # [py<38]
+  entry_points:
+    - manim = manim.__main__:main
+    - manimce = manim.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry >=1.0.10
+    - poetry-core
+  run:
+    - backports.cached-property >=1.0.1,<2.0.0  # [py<38]
+    - click >=7.2
+    - click-default-group >=1.2.2,<2.0.0
+    - cloup >=0.13.0,<0.14.0
+    - colour >=0.1.5,<0.2.0
+    - decorator >=5.0.7,<6.0.0
```

Comment:
Maybe you should also add `ffmpeg` as a dependency since it is used by manim?

Reply:
Yeah I should add it, I forgot it :)